webhooks/php/webhook_callback.php

```diff
@@ -1,10 +1,36 @@
 <?php
-// Read more on https://talkjs.com/docs/Webhooks/Getting_Started.html
-// Parse the request body as JSON
+// This is a vanilla PHP example to read a webhook and validate its integrity. 
```

Comment:
nitpick: I find the wording here slightly weird. I'd change it to

> This is a vanilla PHP example of how to receive webhook requests and validate the signature, so you can be certain that the request was sent by TalkJS.

Reply:
Great point, taking over your suggestion verbatim.